test/drawing_test.dart

```diff
@@ -70,10 +70,10 @@ void main() {
       final GYWIcon icon = GYWIcons.left.icon;
 
       final GYWDrawing drawing = IconDrawing(
-        icon,
+        icon: icon,
         left: 120,
         top: 220,
-        color: Colors.green,
+        color: Colors.green.shade500,
```

Comment:
Why did you change the color ? You don't like the basic green ? :laughing:

Reply:
Actually, `Colors.green` is a family of colors, and its type is `MaterialColor`. `Colors.green.shade500` is a particular Color. The test was failing on my machine because of this.

Reply:
Oh, I know why it fails now and didn't fail before. Apparently, this is false:
`expect(Colors.green, Color(Colors.green.value));`

Before, we had our own equality operator on IconDrawings that compared color.value, not color directly, so we were bypassing the equality operator of Color.

Freezed just compares the color instances, and a MaterialColor is not a Color.

Reply:
I don't think it's necessarily a problem imo. I don't think anybody will realistically serialize, deserialize, and then compare objects like we do in tests.

Reply:
At first, I would agree with you, but after a bit of thinking, shouldn't we store the color value in hex and have a getter with the material color then ? I'm scared of users using the equality operator and not knowing that.

Reply:
Sure, we can do that. It's true that nobody will know the reason the equality fails.